bucket/kotatogram.json

```
@@ -0,0 +1,40 @@
+{
+    "version": "1.4.8",
+    "description": "Experimental Telegram Desktop fork.",
+    "homepage": "https://kotatogram.github.io/",
+    "license": {
+        "identifier": "GPL-3.0 (OpenSSL exception)",
+        "url": "https://github.com/kotatogram/kotatogram-desktop/blob/dev/LICENSE"
+    },
+    "architecture": {
+        "64bit": {
+            "url": "https://github.com/kotatogram/kotatogram-desktop/releases/download/k1.4.8/1.4.8-win64-portable.zip",
+            "hash": "691f09a6d50de6e0a94bba8a6ef18d5b4ffc0e2cf3a849b584d8cbb3543f81a5"
+        },
+        "32bit": {
+            "url": "https://github.com/kotatogram/kotatogram-desktop/releases/download/k1.4.8/1.4.8-win32-portable.zip",
+            "hash": "32a8d3fd60de7ffa85bc7488b7fe3b04328eb367472b79b3ae31e40885fe1553"
+        }
+    },
+    "shortcuts": [
+        [
+            "Kotatogram.exe",
+            "Kotatogram"
+        ]
+    ],
+    "persist": "tdata",
+    "checkver": {
+        "github": "https://github.com/kotatogram/kotatogram-desktop",
+        "re": "Version ([\\d.]+)"
```

```suggestion
        "regex": "Version ([\\d.]+)"
```

We use `regex` now. **re** is only kept for backward compatibility.